backend/dataall/api/Objects/Tenant/resolvers.py

```diff
@@ -32,3 +38,95 @@ def list_tenant_groups(context, source, filter=None):
             data=filter,
             check_perm=True,
         )
+
+
+def update_ssm_parameter(context, source, name: str = None, value: str = None):
+    current_account = SessionHelper.get_account()
+    region = os.getenv('AWS_REGION', 'eu-west-1')
+    print(value)
```

Comment:
Is it for debugging?

Reply:
It stores the dashboard Id and the VPC connection Id. The other alternative was to use the RDS metadata database, but as they are single key-value parameters I opted to store them in the Parameter store